### PR TITLE
Update Testnet rewards doc

### PR DESCRIPTION
**Pull request by mrkara**
- Add longevity description
    - Add more explicit active bid requirement
    - Update the network weight threshold
    - Update the uptime block buffer to 4
    - Remove refs to DxD on the requirements page
    - Extend the validator FAQ
    
    These have been well-documented in various places,
    and communicated through the official comms channel
    on Telegram before. Although we haven't got any complaints
    about anyone not knowing these rules, based on community feedback,
    we are adding here explicitly to make it more accessible
    and easier to notice for the newcomers.

### CODE REVIEW

File: docs/testnet-rewards.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# DEVxDAO Casper Testnet Rewards
+# Casper Testnet Rewards
 
 ## Introduction
 
-Thank you for participating in the DEVxDAO's Casper Testnet program. Your voluntary participation in this Testnet is enabling 
+Thank you for participating in the Casper Testnet program. Your voluntary participation in this Testnet is enabling
```
Comment (mssteuer) [inline docs/testnet-rewards.md:5]: Let's update these references not just here, but also in the footer that's appended to the Testnet related pages

**Review by mssteuer — CHANGES_REQUESTED** [review]

File: docs/testnet-rewards.md [hunk 2]
```diff
@@ -32,18 +32,28 @@ The basic scoring algorithm is:
...
 100 * uptime_percentage
 ```
-where `uptime_percentage` is defined as your node running within 1 block from the "tip" of the blockchain. Your node is regularly scanned
+where `uptime_percentage` is defined as your node running within 4 blocks from the "tip" of the blockchain. Your node is regularly scanned
 on port `8888` to check your block height, so if your port `8888` is closed, you are automatically not considered "up". `uptime_percentage`
 is calculated on a 24-hour basis, during each UTC day. 
 
+#### Longevity Score
+
+The he longevity score is the cumulative daily score of a node since the last fault. Any day with a score below 90 is considered a fault.
```
[the remark this answers is not in the document]
Comment (mrkara) [reply docs/testnet-rewards.md]: Fixed.

File: docs/testnet-rewards.md (hunk 1 above)
Comment (mrkara) [reply docs/testnet-rewards.md:5]: Done.

**Review by mssteuer — APPROVED** [review]
Looks good now, @mrkara. Merge it after the community has had a chance to provide feedback, ie. around 12PM CET on July 4th?

**Comment by mrkara** [comment]
No further suggestions from the community. Merging now.

**Pull request merged**